pymilvus/client/grpc_handler.py

```diff
@@ -654,7 +654,7 @@ def wait_for_creating_index(self, collection_name, index_name, timeout=None, **k
             end = time.time()
             if timeout is not None:
                 if end - start > timeout:
-                    raise MilvusException(message="CreateIndex Timeout")
+                    raise MilvusException(message="CreateIndex Timeout, collection " + collection_name + ", index_name " + index_name)
```

Comment:
can be simplified to:
`f"create index timeout, collection: {collection_name}, index_name: {index_name}"`